Cargo.toml

```diff
@@ -186,7 +186,7 @@ web-time = "1.1"
 wgpu = "22.0"
 winapi = "0.3"
 window_clipboard = "0.4.1"
-winit = { git = "https://github.com/iced-rs/winit.git", rev = "254d6b3420ce4e674f516f7a2bd440665e05484d" }
```

Comment:
I don't know what happened here, none of the commits from iced/master should have my name as the committer...

Reply:
Specifically https://github.com/pop-os/iced/pull/186/commits/c75a4cf40901ded52d80be5eb1a64734a5392122 if that didn't show up with my comment